test/test.js

```diff
@@ -0,0 +1,5 @@
+const sum = require('../bad')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/masterqwerty/sca-testing/pullRequest?prid=7714369)
```suggestion
const sum = require("../bad")
```